src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -2577,6 +2583,16 @@ ExitInfo ExecutorWorker::handleOpsAlreadyExistError(SyncOpPtr syncOp, ExitInfo o
             _syncPal->setRestart(true);
             return removeDependentOps(syncOp);
         }
+
+        // Check if we got the read right on the local item
+        IoError ioError = IoError::Unknown;
+        bool exist = false;
+        IoHelper::checkIfPathExists(_syncPal->localPath() / relativeLocalPath, exist, ioError);
+        if (ioError == IoError::AccessDenied) {
+            LOGW_DEBUG(_logger, Utility::formatSyncPath(relativeLocalPath) << L"has no read access, converting already exist error in file access error.");
```

Comment:
```suggestion
            LOGW_DEBUG(_logger, Utility::formatSyncPath(relativeLocalPath) << L"has no read access, converting 'already exist error' to 'file access error'.");
```

Reply:
I am just waiting the PR including toString(ExitInfo) and I will directly log an exit info (https://github.com/Infomaniak/desktop-kDrive/pull/396)

Reply:
Done here: [aec13c9](https://github.com/Infomaniak/desktop-kDrive/pull/375/commits/aec13c96a0c87cde95baee09ee6d50ad02f486d1)